BotRoyaleUnity/Assets/Scripts/PartHandler.cs

```diff
@@ -1,6 +1,9 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnitySocketIO;
```

Comment:
Preferably we dont need to use this library anywhere in our project except for the wrapper class SocketConnectionHandler